RecoBTag/CTagging/python/candidateNegativeCombinedSecondaryVertexSoftLeptonCvsLComputer_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoBTag.CTagging.candidateCombinedSecondaryVertexSoftLeptonCvsLComputer_cfi import *
+
+candidateNegativeCombinedSecondaryVertexSoftLeptonCvsLComputer = candidateCombinedSecondaryVertexSoftLeptonCvsLComputer.clone()
```

Comment:
you will use less letters if the modified values are passed as arguments in the clone method
```python
candidateNegativeCombinedSecondaryVertexSoftLeptonCvsLComputer = candidateCombinedSecondaryVertexSoftLeptonCvsLComputer.clone(
    vertexFlip = True,
    trackFlip = True,
....
)
```